eslint/README.md

```diff
@@ -0,0 +1,88 @@
+# [ESL](../../../) ESL migration support
+
+Version: *4.12.0*
+
+Authors: *Natalia Smirnova, Alexey Stsefanovich (ala'n)*.
+
+<a name="intro"></a>
+
+This article aims to assist the smooth migration process from older versions of ESL to the latest version. To support this transition, we have developed an ESLint plugin designed specifically for ESL library. This plugin targets deprecated features and aliases within the library, offering a seamless means of identifying these outdated elements. Additionally, it suggests suitable replacements, thereby ensuring a hassle-free upgrade to the newest version of ESL.
+
+### Installation
+To use custom ESLint plugin, you need to install it as a development dependency in your project. You can do this using npm:
```

Comment:
Please add a couple of words for the first step regarding eslint pre requirement + expected minimum version of eslint